chapters/7-publication.tex

```diff
@@ -642,8 +605,8 @@ \subsection{Releasing a reproducibility package}
   \url{https://codeocean.com}}
 offer to store both code and data in one repository,
 and also provide an online workspace in which others can execute and modify your code
-without having to download your tools and match your local environment
-when packages and other underlying software may have changed since publication.
+without having to download your tools and match your local environment. 
+This is particularly useful over time, as packages and other underlying software may have changed since publication.
 In a similar spirit, attempts to create ``metadata archives''
 that link all the different materials, releases, and publications
 relating to specific research projects or studies
```

Comment:
This seems very broad, I'm not sure I understand what it is referring to.

Reply:
@bbdaniels or @mariaruth to add an example

Reply:
Removed in be8cabb68b76e0b552815f6f60f7a317c9b993fd